bpf/kmesh/ads/include/kmesh_common.h

```diff
@@ -9,12 +9,12 @@
 #include "config.h"
 #include "core/address.pb-c.h"
 
-#define BPF_LOGTYPE_LISTENER      BPF_DEBUG_OFF
-#define BPF_LOGTYPE_FILTERCHAIN   BPF_DEBUG_OFF
-#define BPF_LOGTYPE_FILTER        BPF_DEBUG_OFF
-#define BPF_LOGTYPE_CLUSTER       BPF_DEBUG_OFF
-#define BPF_LOGTYPE_ROUTER        BPF_DEBUG_OFF
-#define BPF_LOGTYPE_ROUTER_CONFIG BPF_DEBUG_OFF
+#define BPF_LOGTYPE_LISTENER      BPF_DEBUG_ON
+#define BPF_LOGTYPE_FILTERCHAIN   BPF_DEBUG_ON
+#define BPF_LOGTYPE_FILTER        BPF_DEBUG_ON
+#define BPF_LOGTYPE_CLUSTER       BPF_DEBUG_ON
+#define BPF_LOGTYPE_ROUTER        BPF_DEBUG_ON
+#define BPF_LOGTYPE_ROUTER_CONFIG BPF_DEBUG_ON
 #define BPF_LOGTYPE_COMMON        BPF_DEBUG_OFF
```

Comment:
![image](https://github.com/user-attachments/assets/2d6626ee-a8d2-4f40-980c-6fee0f249f06)
In debug level, this log will be printed many times, but it does not provide useful information, So I think it would be removed,  and then the switch of BPF_LOGTYPE_COMMON can be changed to BPF_DEBUG_ON.

Reply:
Will try